packages/core/src/components/Slider.tsx

```diff
@@ -122,8 +122,6 @@ const styles = StyleSheet.create({
     height: 40,
     flexDirection: "row",
     alignItems: "center",
-    marginLeft: 12,
-    marginRight: 12,
```

Comment:
I think the piece left, according to the ticket, would be to make these a part of Seed Data so that they are user-configurable. That would mean changing the `layout` object down on line 139 to be:
```js
layout: {
  marginLeft: 12,
  marginRight: 12,
}
```

Reply:
done